base/Terminals.jl

```diff
@@ -101,8 +101,8 @@ end
 mutable struct TTYTerminal <: UnixTerminal
     term_type::String
     in_stream::Base.TTY
```

Comment:
Based on the methods defined here, it's only intended for use where the *output* stream is a TTY. But there's no guaranteed relationship between between any of being tty device, having the Base.TTY type, and understanding the ANSI / VT100 codes used here. A more accurate name would be something like ANSIOutputTerminal or VT100OutputTerminal. `_start` just uses the detected type as a heuristic for selecting the right display mechanism.